app/models/micropost.rb

```diff
@@ -0,0 +1,15 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  validates :user_id, presence: true
```

Comment:
Mình xin lỗi viết tiếng anh rồi nhé c tại vì dịch sang tiếng việt hơi khó hiểu :) Nếu vẫn sai thì c chỉ cho mình vs:
- With a model the object just has to be present to make the record valid. It does not have to be persisted yet. 
- With the id, the validation ensures that the object is persisted before the association is valid. So the point of validating presence of association is to ensure the referenced object really exist, and we do need to create a separate user object.